### ISSUE TITLE
"Spin on Kubernetes" page loses TOC expansion

### ISSUE THREAD

**Issue by itowlson** [fermyon/developer#1304]
We normally expand the TOC section containing the current page, and highlight that page:

![image](https://github.com/fermyon/developer/assets/865538/29035bfb-b532-41ab-bbb1-f4130bce1b89)

But for https://developer.fermyon.com/spin/v2/kubernetes this doesn't work: the TOC collapses and there is no "you are here" highlight:

![image](https://github.com/fermyon/developer/assets/865538/a141fadb-4d76-4857-a0e5-aa76a086d110)

Using MS Edge on Windows 11.

**Comment by karthik2804** [comment]
It needs this 
https://github.com/fermyon/developer/pull/1134

**Comment by itowlson** [comment]
@karthik2804 Thanks! That PR is still in draft after 4 months: how much work would be needed to unblock it?

**Comment by karthik2804** [comment]
Probably some 10 minutes of just typing the same thing over and over. I'll get to it tomorrow.

**Comment by itowlson** [comment]
Sounds like a hoot - I can see why it stayed in draft.  Thanks!

**Comment by itowlson** [comment]
#1134 got closed as "no longer needed."  Although the old page linked above still has this issue, the current version is fixed, which suffices to stick a fork in it.
